### PR TITLE
Empty only #118

### PR DESCRIPTION
**Pull request by rupertford**
I've made the changes (which were small) and moved the associated tests into the fortran2003 directory, tidied etc. I think we're ready for first review.

### CODE REVIEW

**Comment by rupertford** [comment]
Thanks to @TeranIvy for volunteering to review.

File: src/fparser/two/Fortran2003.py [hunk 1]
```diff
@@ -7296,49 +7297,68 @@ def match(string):
...
+                               empty string
+        :raises InternalError: if items array[3] is 'None' as it should be \
+                               a string
         '''
+        if len(self.items) != 5:
+            raise InternalError(
+                "Use_Stmt.tostr(). 'Items' should be of size 5 but found "
+                "'{0}'.".format(len(self.items)))
+        if not self.items[2]:
+                raise InternalError("Use_Stmt.tostr(). 'Items' entry 2 should "
```
[the remark this answers is not in the document]
Comment (rupertford) [reply src/fparser/two/Fortran2003.py]: Oops, I missed that. Thanks.

File: src/fparser/two/Fortran2003.py [hunk 2]
```diff
@@ -7296,49 +7297,68 @@ def match(string):
...
+                                    "be a module name but it is empty")
+        if self.items[3] is None:
+                raise InternalError("Use_Stmt.tostr(). 'Items' entry 3 should "
```
[the remark this answers is not in the document]
Comment (rupertford) [reply src/fparser/two/Fortran2003.py]: And again.

File: src/fparser/two/Fortran2003.py [hunk 3]
```diff
@@ -7296,49 +7297,68 @@ def match(string):
...
         name = Module_Name(name)
-        line = line[i+1:].lstrip()
+        line = line[position+1:].lstrip()
         # Missing 'ONLY' specification after 'USE Module_Name,'
         if not line:
             return
         if line[:4].upper() == 'ONLY':
             line = line[4:].lstrip()
+            if not line:
+                # Expected ':' but there is nothing after the 'ONLY'
```
[the remark this answers is not in the document]
Comment (rupertford) [reply src/fparser/two/Fortran2003.py]: I've hopefully made it more clear :-)

**Comment by rupertford** [comment]
I've hopefully addressed the reviewers comments. Back to @TeranIvy for second review.

**Comment by TeranIvy** [comment]
Looks good and `pylint` warnings are fixed. I had to refresh the branch prior to merge to master so I can then edit changelog again and then merge.

**Pull request merged**